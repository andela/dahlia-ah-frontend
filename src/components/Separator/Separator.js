import React from 'react';
import './Seperator.scss';

const Separator = () => (
  <div>
    <hr />
    <span className="overlay">OR</span>
  </div>
);

export default Separator;
