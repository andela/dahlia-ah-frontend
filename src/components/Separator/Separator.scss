.overlay {
    position: absolute;
    transform: translate(-40px, -20px);
    z-index: 200;
    display: inline-block;
    padding: 0 30px;
    background: #fafafa;
}