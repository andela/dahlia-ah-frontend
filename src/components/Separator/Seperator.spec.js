import React from 'react';
import { configure, shallow } from 'enzyme';

import Adapter from 'enzyme-adapter-react-16';
import Separator from './Separator';

configure({ adapter: new Adapter() });

describe('Seperator component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <Separator />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
