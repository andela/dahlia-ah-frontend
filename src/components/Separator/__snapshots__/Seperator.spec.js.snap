// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seperator component renders correctly 1`] = `ShallowWrapper {}`;
